geode-core/src/main/java/org/apache/geode/cache/client/PoolFactory.java

```diff
@@ -220,6 +220,14 @@
    * @since GemFire 6.5
    */
   boolean DEFAULT_MULTIUSER_AUTHENTICATION = false;
+  /**
+   * The default value for whether to request locator internal address.
```

Comment:
Slightly more detail would be good here, explaining when exactly the locator internal address would be requested if this was set to true.